### PR TITLE
[app] Update simple map camera

### PR DESCRIPTION
**Pull request by Chaoba**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [x] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog>[app] Update simple map camera</changelog>`.

### Summary of changes
In this pr:

- Update simple map camera align with gl js https://docs.mapbox.com/mapbox-gl-js/example/simple-map/
- Update title and description for add a marker
- Update title for simple map

### CODE REVIEW

File: app/src/main/res/layout/activity_simple_map.xml [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+    xmlns:mapbox="http://schemas.android.com/apk/res-auto"
     android:id="@+id/mapView"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".examples.SimpleMapActivity" />
\ No newline at end of file
+    mapbox:mapbox_cameraTargetLat="40"
+    mapbox:mapbox_cameraTargetLng="-74.5"
+    mapbox:mapbox_cameraZoom="9"
+    tools:context=".examples.SimpleMapActivity" />
```
Comment (kiryldz) [inline app/src/main/res/layout/activity_simple_map.xml:11]: Guess we could remove those tools.

**Review by kiryldz — APPROVED** [review]
LGTM, also add `skip changelog` label.

File: app/src/main/res/layout/activity_simple_map.xml (hunk 1 above)
Comment (pengdev) [reply app/src/main/res/layout/activity_simple_map.xml:11]: I think this xml is also used in `SimpleMapActivity` and probably we could use a different xml for `AddOneMarkerSymbolActivity` if we want to add these properties?

**Review by pengdev — APPROVED** [review]
LGTM, with one nit.

File: app/src/main/res/layout/activity_simple_map.xml (hunk 1 above)
Comment (Chaoba) [reply app/src/main/res/layout/activity_simple_map.xml:11]: Yes, need to add a new one for AddOneMarkerSymbolActivity

Comment (Chaoba) [reply app/src/main/res/layout/activity_simple_map.xml:11]: This layout file is used in many places, need to revert the changes on it.

**Pull request merged**